app/javascript/packs/webauthn-setup.js

```diff
@@ -22,9 +38,7 @@ function webauthn() {
         document.getElementById('client_data_json').value = result.clientDataJSON;
         document.getElementById('webauthn_form').submit();
       })
-      .catch(function (err) {
-        window.location.search = `?error=${err.name}`;
-      });
+      .catch((err) => reloadWithError(err.name));
```

Comment:
#5662 added a new [`addSearchParams` utility](https://github.com/18F/identity-idp/blob/78288083430385c2bfe8ec8a609b139d2ff4262a/app/javascript/packages/url/index.js#L1-L29), which could be useful here:

```suggestion
      .catch((error) => {
        location.search = addSearchParams(location.search, { error: error.name });
      });
```